pkg/loki/modules.go

```diff
@@ -473,3 +499,16 @@ var modules = map[moduleName]module{
 		deps: []moduleName{Querier, Ingester, Distributor, TableManager},
 	},
 }
+
+// ActiveIndexType type returns index type which would be applicable to metrics that would be pushed starting now
+// Note: Another periodic config can be applicable in future which can change index type
+func ActiveIndexType(cfg chunk.SchemaConfig) string {
```

Comment:
I know you want to add test later, and you should write a test for that function, doesn't look like this would change and it seems easy to write.